src/UserHandler.cpp

```diff
@@ -0,0 +1,241 @@
+//
+// Created by idanto on 01/01/2020.
+//
+
+#include "../include/UserHandler.h"
+
+
+UserHandler::UserHandler(ConcurrentDataQueues& queues, ClientInventory& inventory) :
+        _queues(queues), _inventory(inventory), _actionFrameCommandMap(initActionFrameCommandMap()) {}
+
+
+void UserHandler::run() {
+    while (true) {
+        const short bufSize = 1024;
+        char buf[bufSize];
+        std::cin.getline(buf, bufSize);
+        std::string line(buf);
+        parseLine(line);
+    }
+}
+
+UserActionsEnum UserHandler::findCmd(std::string line) {
+    std::string cmd = line.substr(0, line.find(" "));
+    if (cmd == "login") {
```

Comment:
why would you find a " " in the line? wouldn't it be \n?

Reply:
nope, its from cin